### PR TITLE
[Docs] Update introduction and fix issues

### PR DESCRIPTION
**Pull request by linyq17**
## Modification
1. add an introduction to docs.
2. fix links and typos.
3. update docs apis config.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ MMFewShot 是一款基于 PyTorch 的少样本学习代码库，是 [OpenMMLab](
 请参考[安装文档](docs/install.md)进行安装和参考[数据准备](tools/data/README.md)准备数据集。
 
 ## 快速入门
-
-请参考[使用教程](docs/get_started.md)获取MMFewShot的基本用法。
+如果初次了解少样本学习，你可以从[Learn the basics](docs/intro.md)开始了解少样本学习的基本概念和 MMFewShot 的框架。
```
Comment (jshilong) [inline README_zh-CN.md:87]: Please use Chinese all the time in Chinese documentation

File: docs/intro.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Learn the Basics
+
+This chapter introduces you to the basic conception of few shot learning, and the framework of
```
Comment (jshilong) [inline docs/intro.md:3]: Should we modify all `few shot learning` to `Few-shot learning` ?

Comment (linyq17) [reply docs/intro.md:3]: we use `few shot learning` and `MMFewShot` as default.

File: docs/intro.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+MMFewShot, and provides links to detailed tutorials about MMFewShot.
+
+## What is Few Shot Learning
+
+### Problem definition
+Few-shot learning aims at generalizing to new tasks based on a limited number of samples using prior knowledge.
+The prior knowledge usually refers to a large scale training set which has many classes and samples,
```
Comment (jshilong) [inline docs/intro.md:10]: that has many

File: docs/intro.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+while the samples in new tasks are never seen in the training set.
+For example, in few shot image classification, a pre-trained model only can see
+five bird images (each class has one image and doesn't exist in the pretrained dataset) and
+predict the bird in query image belongs to which class.
+And in few shot detection, a detector needs to detect the new categories based on a few instances.
+
+In summary, few shot learning focus on two aspects:
+- how to embed prior knowledge into models (pre-train with large scale dataset)
+- how to transfer knowledge to adapt new tasks (learn on a few labeled samples).
```
Comment (jshilong) [inline docs/intro.md:19]: adapt to new tasks

File: docs/intro.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+
+<div align=center>
+<img src="https://github.com/open-mmlab/mmfewshot/blob/main/resources/demo.png?raw=true" width=80%/>
+</div>
+
+### Terminologies in few-shot learning
+- Training set: every class in the training set has many samples, and it is big enough for learning a deep neural network.
```
Comment (jshilong) [inline docs/intro.md:27]: for training a deep neural network.

File: docs/intro.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+- Support set: a small set of labeled images and all the classes do not exist in training set.
```
Comment (jshilong) [inline docs/intro.md:28]: in the training set.

File: docs/intro.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+- Query set: unlabeled images to predict and share the same classes with support set.
+
+- N way K shot: the support set setting
```
Comment (jshilong) [inline docs/intro.md:31]: It means support images contain N classes and each class samples K images， so there will be N×K support images in each batch.

File: docs/intro.md (hunk 2 above)
Comment (jshilong) [reply docs/intro.md:3]: L9

Few-shot learning aims at generalizing to new tasks based on a limited number of 
 samples using prior knowledge.

File: docs/intro.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+   - N-way means the support set has k classes.
+   - K-shot means every class has n samples.
+
+
+### Evaluation
+#### Few shot classification
+Meta test: randomly sample different tasks (N way K shot + Q query images)
+and calculate the accuracy mean and std of the prediction on query images.
```
Comment (jshilong) [inline docs/intro.md:39]: and calculate the mean and std of prediction accuracy on query images.

File: docs/intro.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+#### Few shot detection
+The novel class performance (mAP or AP50) on test set.
```
Comment (jshilong) [inline docs/intro.md:41]: ?

File: docs/intro.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+
+
+### The basic pipeline for few shot learning
+We will introduce a simple baseline for all the few shot learning task to further illustrate how few shot learning work.
```
Comment (jshilong) [inline docs/intro.md:46]: tasks

File: docs/intro.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+The most obvious pipeline is fine-tuning.
+It usually consists of two steps: train a model on a large scale dataset and then fine-tune on few shot data.
+For image classification, we first pretrain a model with training set using cross-entropy loss, and then
+we can transfer the backbone and fine tune a new classification head.
+For detection, we can first pretrain a faster-rcnn on training set, and
+then fine tune a new bbox head on a few instances to detect novel class.
```
Comment (jshilong) [inline docs/intro.md:52]: detect the novel class.

File: docs/intro.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+MMFewShot is the first toolbox that provides a framework for unified implementation and evaluation of few shot classification and detection methods,
+and below is its whole framework:
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/15669896/143182168-e2b0a3b8-4dce-4e44-b134-a2577c8290c5.png" width=80%/>
+</div>
+
+MMFewShot consists of 4 main parts, `datasets`, `models`, `core` and `apis`.
+
+- `datasets` is for datasets loading and data augmentation. In this part,
```
Comment (jshilong) [inline docs/intro.md:66]: datasets` is for data loading and data augmentation

File: docs/intro.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+we support various datasets for classification and detection algorithms,
+useful data augmentation transforms in `pipelines` for pre-processing image
```
Comment (jshilong) [inline docs/intro.md:68]: flexible

File: docs/intro.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+and  `datasetswrappers` for data sampling.
+
+- `models` is the most vital part containing models and loss functions.
```
Comment (jshilong) [inline docs/intro.md:71]: `models` contains models and loss functions.

File: docs/intro.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+- `core` provides evaluation tools and customized hooks for model training and evaluation.
+
+- `apis` provides high-level APIs for models training, testing, and inference,
```
Comment (jshilong) [inline docs/intro.md:75]: , to .

File: model_zoo.yml [hunk 16]
```diff
@@ -24,26 +24,26 @@ Import:
   - configs/classification/relation_net/mini_imagenet/metafile.yml
   - configs/classification/relation_net/tiered_imagenet/metafile.yml
   - configs/detection/attention_rpn/coco/metafile.yml
-  - configs/detection/attention_rpn/voc_split1/metafile.yml
-  - configs/detection/attention_rpn/voc_split2/metafile.yml
-  - configs/detection/attention_rpn/voc_split3/metafile.yml
+  - configs/detection/attention_rpn/voc/split1/metafile.yml
```
Comment (hellock) [inline model_zoo.yml:27]: model_zoo.yml is unnecessary and duplicated. model-index.yml should be kept.

File: README.md [hunk 17]
```diff
@@ -88,9 +88,26 @@ Please refer to [install.md](docs/install.md) for installation of mmfewshot and
 
 ## Getting Started
 
-Please see [getting_started.md](docs/get_started.md) for the basic usage of mmfewshot.
+If you're new of few shot learning, you can start with [Learn the basics](docs/intro.md).
```
Comment (jshilong) [inline README.md:91]: if you are new to

File: docs/intro.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+MMFewShot, and provides links to detailed tutorials about MMFewShot.
+
+## What is Few Shot Learning
+
+### Problem definition
+Few shot learning aims at generalizing to new tasks based on a limited number of samples using prior knowledge.
+The prior knowledge usually refers to a large scale training set that has many classes and samples,
+while the samples in new tasks are never seen in the training set.
+For example, in few shot image classification, a pre-trained model only can see
+five bird images (each class has one image and doesn't exist in the pretrained dataset) and
```
Comment (jshilong) [inline docs/intro.md:13]: each class has an image which doesn't exist in the pretrained dataset

File: docs/intro.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+predict the bird in query image belongs to which class.
```
Comment (jshilong) [inline docs/intro.md:14]: predict the class of bird in query image

File: docs/intro.md [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+And in few shot detection, a detector needs to detect the new categories based on a few instances.
```
Comment (jshilong) [inline docs/intro.md:15]: The expression of  `and` is not fluent enough

**Review by jshilong — APPROVED** [review]
LGTM
